src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/CSharpCodeWriter.cs

```diff
@@ -302,9 +362,10 @@ public CSharpCodeWriter WriteEndMethodInvocation(bool endLine)
         }
```

Comment:
Is the WriteStartMethodInvocation with generic parameters used a lot? It's a rough API and thinking it'd make more sense to not have it.